packages/databricks-vscode/src/bundle/BundleCommands.ts

```diff
@@ -61,6 +65,9 @@ export class BundleCommands implements Disposable {
         await this.refreshRemoteState();
     }
 
+    private deployMutex = new Mutex();
+
+    @Mutex.synchronise("deployMutex")
```

Comment:
If someone clicks on the deploy button 10 times, will we just deploy the bundle 10 times one after another? Should we throttle here instead? Similar questions for the other command above